activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -268,6 +274,46 @@ public void createProcessInstanceAndValidateHappyPath() {
         assertThat(RuntimeTestConfiguration.discardImageConnectorExecuted).isEqualTo(false);
     }
 
+    @Test
+    public void createLinkEventProcessInstanceAndValidateHappyPath() {
+        //when
+        ProcessInstance linkProcess = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(LINK_PROCESS)
+            .withVariable("expectedKey",
+                true)
+            .build());
+
+        assertThat(RuntimeTestConfiguration.completedProcesses).contains(linkProcess.getId());
+    }
+
+    @Test
+    public void createLinkEventProcessInstanceWithInclusiveGatewayAndValidateHappyPath() {
+        //when
+        ProcessInstance linkProcess = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(LINK_PROCESS_WITH_GATEWAY)
+            .withVariable("expectedKey",
+                true)
+            .build());
+        TaskQuery query = taskService.createTaskQuery().orderByTaskName().asc();
+        var tasks= query.list();
+        assertThat(tasks).hasSize(2);
+        assertThat(tasks.get(0).getName()).isEqualTo("task 1");
+        assertThat(tasks.get(1).getName()).isEqualTo("task 2");
```

Comment:
This can be simplified to:
```
assertThat(tasks)
            .extracting(org.activiti.engine.task.Task::getName)
            .containsExactly("task 1", "task 2");
```

`containsExactly` is already verifying the size and the order. Same for the next assertions.